lib/utils.pm

```diff
@@ -1304,6 +946,14 @@ sub arrays_differ {
     return 0;
 }
 
+sub ensure_serialdev_permissions {
+    my ($self) = @_;
+    # grant user permission to access serial port until next reboot
+    # and get permanent user permission to access serial port even if reboot
+    assert_script_run "chown $testapi::username /dev/$testapi::serialdev";
+    assert_script_run "gpasswd -a $testapi::username \$(ls -l /dev/$testapi::serialdev | awk \"{print \\\$4}\")";
```

Comment:
IMHO it is better to add the user to a fixed group (e.g. dialout) to catch if permissions ever change.

Reply:
agreed